src/user.js

```diff
@@ -0,0 +1,27 @@
+class User {
+  constructor(user) {
+    this.id = user.id;
+    this.name = user.name;
+    this.travelerType = user.travelerType;
+  }
+
+  logIn(userName, password) {
+    if (userName ===  traveler/*find function to have nums 1-50*/ && password = 'travel2020') {
+      domUpdates.populateTravelerPage()
+    } else if (userName === 'agency' && password === 'travel2020') {
+      domUpdates.populateAgentPage()
+    } else {
+      domUpdates.loginErrorDisplay()
```

Comment:
This is interesting having your DOM updates happen in your User class file, but I think it makes sense for this particular function. Something to think about: Where in those functions will the fetch request happen? Or will you have the fetch requests happening in the background before the login happens?